.changelog/223.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+all: Remove newline from HostInfo.NativeArchitecture
```

Comment:
I think this is only affecting linux so suggest:

```suggestion
linux: Remove newline from HostInfo.NativeArchitecture value.
```

Reply:
Actually I first noticed it on macOS so it affects that platform too.